newIDE/app/src/ResourcesList/ResourcePreview/Resource3DPreview.worker.js

```diff
@@ -3,6 +3,47 @@
 import * as THREE from 'three';
 import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader';
 
+const isNativeMobileApp = false;
+
+// Copied from Utils/CrossOrigin.js
```

Comment:
In each file, add a "reverse" comment telling "If changed, also change the duplicated implementation in `newIDE/app/src/ResourcesList/ResourcePreview/Resource3DPreview.worker.js`."